conf/machine/hikey.conf

```diff
@@ -17,6 +17,11 @@ MACHINE_FEATURES = "usbhost usbgadget alsa screen wifi bluetooth efi"
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-hikey"
 
+PREFERRED_PROVIDER_virtual/libgles1 ?= "mali450-userland-drm"
+PREFERRED_PROVIDER_virtual/libgles2 ?= "mali450-userland-drm"
+PREFERRED_PROVIDER_virtual/egl ?= "mali450-userland-drm"
```

Comment:
I pushed the recipe into our meta-los layer, moving this code to meta-los as well for now.